leku/src/main/java/com/adevinta/leku/LocationPickerActivity.kt

```diff
@@ -434,6 +405,52 @@ class LocationPickerActivity : AppCompatActivity(),
         }
     }
 
+    private fun createSearchViewTextChangeObservable(): Observable<String> = Observable.create { subscriber: ObservableEmitter<String> ->
+        searchView?.addTextChangedListener(object : TextWatcher {
+            override fun afterTextChanged(s: Editable) {}
+            override fun beforeTextChanged(s: CharSequence, start: Int, count: Int, after: Int) {}
+            override fun onTextChanged(s: CharSequence, start: Int, before: Int, count: Int) {
+                subscriber.onNext(s.toString())
+            }
+        })
+    }.debounce(DEBOUNCE_TIME.toLong(), TimeUnit.MILLISECONDS, AndroidSchedulers.mainThread())
```

Comment:
All our debounces are on the IO thread, I'm not sure of the implications of this being in the mainthread 🤔